src/js/profile/chart.js

```diff
@@ -271,15 +271,15 @@ export class Chart extends Component {
             $(this).on('click', () => {
                 let exportType = $(this).data('id');
                 const downloadFn = {
-                    0: {type: 'csv', fn: self.exportAsCsv},
-                    1: {type: 'excel', fn: self.exportAsExcel},
-                    2: {type: 'json', fn: self.exportAsJson},
-                    3: {type: 'kml', fn: self.exportAsKml},
-                }[index];
-                self.triggerEvent(`profile.chart.download_${downloadFn['type']}`, self);
+                    'csv': self.exportAsCsv,
+                    'excel': self.exportAsExcel,
+                    'json': self.exportAsJson,
+                    'kml': self.exportAsKml,
+                };
+                self.triggerEvent(`profile.chart.download_${exportType}`, self);
 
                 let fileName = self.getChartTitle('-');
-                downloadFn.fn(fileName);
+                downloadFn[exportType](fileName);
```

Comment:
This is the fix included from https://github.com/OpenUpSA/wazimap-ng-ui/pull/416
Reminder to close that PR when this is merged

Reply:
This comment is meant for @adieyal

Reply:
github will actually do that automatically